docs/pages/access-controls/reference.mdx

```diff
@@ -120,7 +120,7 @@ The table below documents the behavior of each option if multiple roles are assi
 | `pin_source_ip` | Enable source IP pinning for SSH certificates. **Note:** IP pinning is currently in Preview mode | Logical "OR" i.e. evaluates to "yes" if at least one role requires session termination |
 | `cert_extensions` | Specifies extensions to be included in SSH certificates | |
 | `create_host_user_mode` | Allow users to be automatically created on a host |  Logical "AND" i.e. if all roles matching a server specify host user creation (`off`, `drop`, `keep`), it will evaluate to the option specified by all of the roles. If some roles specify both `drop` or `keep` it will evaluate to `keep`|
-| `create_db_user_mode` | Allow [database user auto provisioning](../database-access/auto-user-provisioning.mdx) | Logical "OR" i.e. if any role allows database user auto-provisioning, it's allowed |
+| `create_db_user_mode` | Allow [database user auto provisioning](../database-access/auto-user-provisioning.mdx) (`off`, `keep`, `best_effort_drop`) | Logical "OR" i.e. if any role allows database user auto-provisioning, it's allowed |
```

Comment:
Is there a way to describe each possible value here?

Reply:
I've added the same description as the role spec.